libs/positions/src/lib/positions-data-providers.ts

```diff
@@ -135,6 +136,8 @@ export const getMetrics = (
       totalBalance: totalBalance.multipliedBy(10 ** decimals).toFixed(),
       unrealisedPNL: position.unrealisedPNL,
       updatedAt: position.updatedAt || null,
+      productType:
+        market?.tradableInstrument.instrument.product.__typename || '',
```

Comment:
Not sure how it can be done, but when these changes https://github.com/vegaprotocol/vega/pull/9003 will be available on stagnet or testnet it all these types will have to be amend and adjusted to